examples/Thermostat/slice/Thermostat.slice

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ZeroC, Inc.
+
+module Igloo
+
+/// Monitors and controls a residential cool-only AC system.
+/// This interface corresponds to the public API used by client applications.
+interface Thermostat {
```

Comment:
I think we're somehow missing out here by having two interfaces with `changeSetPoint(setPoint: float32)`.  One option is to rename this interface to `ThermoMonitor` and remove the `chagneSetPoint` operations. The implement would then implement `ThermoMonitor` and `ThermoControl`.

Reply:
I discussed this with Joe and I prefer keeping the Thermostat interface as-is.